### ISSUE TITLE
Test RabbitMQ following #3025

### ISSUE THREAD

**Issue by dlebauer** [PecanProject/pecan#3088]
The final task for #3025 was to test that it still works using RabbitMQ. Rather than holding up the PR, I am opening this issue for @robkooper to go back and test that the system still works with RabbitMQ.

We should also address the question:
- is there a way to automatically test this feature using GH actions? 
- does the fact that a PR is blocked by the need for a manual test indicate an undesirable pattern?

**Comment by Aariq** [comment]
Not sure, but this might help with testing: https://github.com/marketplace/actions/rabbitmq-in-github-actions

**Comment by robkooper** [comment]
other option is to bring up the full stack using docker and run a single test through curl. same as the current sipnet test (which runs with local install). See https://github.com/PecanProject/pecan/actions/runs/3893869709/workflow#L143

**Comment by robkooper** [comment]
tested with rabbitmq, it works

**Comment by robkooper** [comment]
do you want to rename this issue or should we open a a new issue that is specific for more complex testing.
